libhttpserver/content_type_utils.go

```diff
@@ -37,11 +37,12 @@ func (w *contentTypeOverridingResponseWriter) overrideMimeType(
 	case strings.HasPrefix(ty, "audio/") ||
 		strings.HasPrefix(ty, "image/") ||
 		strings.HasPrefix(ty, "video/") ||
-		ty == "application/pdf":
+		ty == "application/pdf" ||
+		ty == "text/plain":
```

Comment:
Shouldn't this be `strings.HasPrefix(ty, "text/")`?

And IIRC we need it to always return `text/plain` when it's any type of text.

Reply:
Sure we can do that, Note however that we already do html etc interception before. Will have a revised patch up in 2min